tests/integration_tests/functional/test_serial_io.py

```diff
@@ -145,7 +145,6 @@ def test_serial_dos(test_microvm_with_api):
     # Set up the microVM with 1 vCPU and a serial console.
     microvm.basic_config(
         vcpu_count=1,
-        add_root_device=False,
```

Comment:
While developing this commit I found this was not needed by the test. I included it here, but I could also split it out as a separate commit if that helps.